Automattic/wp-calypso
# Jetpack Products: Add new page for upgrading storage tier of realtime products

#### Changes proposed in this Pull Request

This PR adds a new pricing page for the realtime-only backup products.

Note that this PR does not change the features on the cards. That will be done in a later iteration.

![image](https://user-images.githubusercontent.com/42627630/134545271-fa7a9cda-b87e-4c94-b960-d4e99fb0d85e.png)

#### Testing instructions



1. To start, visit `jetpack.cloud.localhost:3001` and generate an OAuth token if you need one.
2. Visit `jetpack.cloud.localhost:3001/pricing/storage/:siteSlug?flags=jetpack/only-realtime-products`.
3. Do this for a site with each of the tier 1/2 backup/security products.
4. Verify that each page displays correctly, and that the upgrade URLs are all correct. Note that this page is only intended to display for users with tier 1 products, so focus on these. Users with tier 2 products would need to have this page bookmarked to get here, so it should not crash but what is displayed may not be the best option.



Related to 1200412004370260-as-1200532973279628
